src/aap_eda/core/views.py

```diff
@@ -1,10 +1,31 @@
+from django.db import connection
+from django.db.utils import OperationalError
+from rest_framework.permissions import AllowAny
 from rest_framework.response import Response
 from rest_framework.views import APIView
 
 
 class HealthView(APIView):
-    authentication_classes = []
-    permission_classes = []
+    permission_classes = [AllowAny]
 
-    def get(self, _request):
-        return Response("OK")
+    def get(self, request):
+        try:
+            connection.ensure_connection()
+            return Response({"status": "OK"}, status=200)
+        except OperationalError:
+            return Response(
+                {"error": "Database connection failed"}, status=500
```

Comment:
Should we include here also the "status" key for consistency?

Reply:
yep, makes sense!